checker/ignore.go

```diff
@@ -37,17 +38,21 @@ func ProcessIgnoredBackwardCompatibilityErrors(level Level, errs Changes, ignore
 	ignoredErrs := make([]bool, len(errs))
 	for ignoreScanner.Scan() {
 		ignoreLine := strings.ToLower(ignoreScanner.Text())
-
-		ignorePath := ignoreLinePath(ignoreLine)
-		if ignorePath == "" {
-			continue
-		}
+		ignorePath := ""
 
 		for errIndex, err := range errs {
 			if err.GetLevel() != level {
 				continue
 			}
 
+			typeOfChange := fmt.Sprintf("%T", err)
+			if typeOfChange != "checker.ComponentChange" {
```

Comment:
Can we implement this through the `Change` interface instead of adding this `if`?

Reply:
yes, great idea, I'm adding the check for empty ignore paths within the `MatchIgnore`